_pages/design-guidelines.md

```diff
@@ -1,10 +1,12 @@
 ---
-title: Design assets
+title: User Experience
 lead: >
   Integrate with login.gov in a user-friendly manner.
 sidenav:
-  - text: Overview
+  - text: User Experience
     href: "#overview"
+  - text: Sign-in guidelines
+    href: "#signin-guidelines"
```

Comment:
the auto-generated slug is "sign-in-guidelines" (with an extra dash), it needs to be fixed to pass the build

Reply:
Thanks Zach, it's been fixed and pushed